components/script/dom/webgl_validations/tex_image_3d.rs

```diff
@@ -114,7 +114,7 @@ impl WebGLValidator for CommonTexImage3DValidator<'_> {
         }
 
         // GL_INVALID_VALUE is generated if border is not 0 or 1.
-        if self.border != 0 || self.border != 1 {
+        if !(self.border == 0 || self.border == 1) {
```

Comment:
Where does the border==1 check come from? Everything I've read in the WebGL and OpenGL ES specifications only say 0.